src/components/Card.tsx

```diff
@@ -243,7 +243,7 @@ const StepCard = ({
           <StepCardHeader>
             <div>
               {leftHeaderChildren}
-              <div className="question-info">
+              <div className="question-info" role="heading" aria-level={2}>
```

Comment:
seems like it's preferable to just use a heading tag: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA12

Reply:
They cite styling as a reason it's preferable, but the reason I didn't use a heading tag was that I didn't want the style to change.

Reply:
> Native HTML elements have implicit styling and features that won't necessarily be replicated with ARIA equivalents. For instance, heading elements will have specific styling adjustments made when viewing a page in a browser's Reader Mode.

I was more wondering about that second bit but it looks like our iframe doesn't currently render at all in reader mode...